docs/snapcraft-syntax.md

```diff
@@ -0,0 +1,84 @@
+# Syntax of the snapcraft.yaml file
+
+The `snapcraft.yaml` file is the main entry point to create a snap through
+Snapcraft. The main building blocks are parts and each defined part is
+independent from each other. In addition to parts, there are attributes
+that define the metadata for the snap package.
+
+What follows is a list of all the attributes the `snapcraft.yaml` file can
+contain.
+
+ * `name` (string)  
+   The name of the resulting snap.
+ * `version` (string)  
+   The version of the resulting snap.
+ * `vendor` (string)  
+   Vendor in email format as defined in RFC 5322.
+ * `summary` (string)  
+   A 78 character long summary for the snap.
+ * `description` (string)  
+   The description for the snap, this can and is expected to be a longer
+   explanation for the snap.
+ * `config` (string)  
+   Path to the runnable in snap that will be used as the Snappy config
+   interface.
```

Comment:
I personally (a relative newcomer to snappy) am not familiar with this. Have I missed a document, or should this section contain a reference link?